views/forms/TextInput/index.jsx

```diff
@@ -0,0 +1,24 @@
+import TextField from '@material-ui/core/TextField'
+import { Controller } from 'react-hook-form'
+
+const TextInput = (props) => {
+  const { name, control, ...rest } = props
+
+  return (
+    <Controller
+      name={name}
+      control={control}
+      render={({ field }) => (
+        <TextField
+          type="text"
```

Comment:
One would think this would be the default type of this component. Guess not!

Reply:
[6c00bd8](https://github.com/AMP-SCZ/dpdash/pull/332/commits/6c00bd838e1c74c9454f70d74b10a83e0f793886)